### PR TITLE
Improve markdown Formatting of beginners guide

### PR DESCRIPTION
**Pull request by fbehrens**
I just improved the formatting of this markdown document.

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [NA] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [X] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

**Comment by anmenaga** [comment]
I edited description and filled necessary checkboxes of the description template.

File: docs/learning-powershell/powershell-beginners-guide.md [hunk 1]
```diff
@@ -93,42 +94,42 @@ Alias           gmo -> Get-Module
...
 Alias           type -> Get-Content
 …
+```
 
-As you can see "cls" is an alias of Clear-Host.
-Now try it:
+As you can see `cls` is an alias of `Clear-Host`.
 
+Now try it:
+```
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/powershell-beginners-guide.md]: Should be: 
 \```powershell

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by anmenaga** [comment]
@SteveL-MSFT your feedback was addressed; please take a look again.

**Review by anmenaga — APPROVED** [review]

File: docs/learning-powershell/powershell-beginners-guide.md [hunk 2]
```diff
@@ -18,17 +18,17 @@ In this section, you will learn how to
...
+It is shown as `PS C:\>` on  Windows.
 
-**1. Get-Process**: Gets the processes that are running on the local computer or a remote computer.
+1. `Get-Process`: Gets the processes that are running on the local computer or a remote computer.
 
 By default, you will get data back similar to the following:
 
-``` PowerShell
+``` powershell
 PS /> Get-Process
```
Comment (alexandair) [inline docs/learning-powershell/powershell-beginners-guide.md]: Remove a space between backticks and "powershell".

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@fbehrens Please rebase or send in a new commit to trigger `PowerShell-CI-spelling` CI.

**Review by adityapatwardhan — APPROVED** [review]

**Comment by fbehrens** [comment]
Thanks for merging :)

**Comment by iSazonov** [comment]
@fbehrens Thanks for your contribution!

**Pull request merged**